### ISSUE TITLE
it does not update

### ISSUE THREAD

**Issue by gerroon** [igelbox/blender-retarget#7]
Hi

Thanks for the add-on. This seems to work  however it does not update during the playback. I have to go into the edit mode in an out to see it. Is there a way to make it work in realtime playback?

I am using 2.83 on Windows

**Comment by igelbox** [comment]
Yeah, it seems my 2.8 hack isn't work since 2.82 (
I need time to research if there's something blender guys invented to control the depsgraph.

**Comment by igelbox** [comment]
Well, it happened I spent some time researching and figured out 1.5 automatic solutions:
* 1 - Add one more variable and point it at the source object (that would make dependency and force Blender to update the target pose in time), but, it's like one more hack and doesn't work in 2.82 (only >= 2.83)
* 1/2 - Use `bpy.props.CollectionProperty` somehow to represent the Target object as part of the Source object. This would allow to link the Target object drivers to Source object and therefore make dependency. This seems like a much robust/reliable solution, but, I'm not sure I can implement it using `bpy.props` capabilities. I tried to achieve it using Python features (`__getattr__` and `__getitem__`) but Blender barks the expression using such objects is invalid.
